sdk/core/src/debug/java/com/klaviyo/core/Logger.kt

```diff
@@ -0,0 +1,31 @@
+package com.klaviyo.core
+
+import com.klaviyo.core.config.Log
+import com.klaviyo.core.config.NetworkRequest
+import java.lang.Exception
+
+internal object Logger : Log {
+    override fun debug(output: String) {
+        println(output)
```

Comment:
Will these use android logging at a later date?

Reply:
what do you mean by android logging? there's Logger but thats just from `java.utils`

Reply:
I put `println` in as a placeholder, I'm going to flesh this out with a snippet we were going to copy from timber that includes stack trace, and I think it even had a way to color the output too?

Reply:
Android has built in logging capabilities. I think timber was a wrapper around it. https://developer.android.com/reference/android/util/Log